pumpkin/src/command/mod.rs

```diff
@@ -84,7 +84,7 @@ pub fn default_dispatcher<'a>() -> Arc<CommandDispatcher<'a>> {
     dispatcher.register(cmd_teleport::init_command_tree());
     dispatcher.register(cmd_give::init_command_tree());
     dispatcher.register(cmd_list::init_command_tree());
-    dispatcher.register(cmd_clear::init_command_tree());
```

Comment:
I'm not sure whether this just happens because the clear command was merged after you created this pr, but this line shouldn't be removed

Reply:
Fixed ✅